federation/cmd/federation-controller-manager/app/options/options.go

```diff
@@ -39,6 +39,8 @@ type ControllerManagerConfiguration struct {
 	FederationName string `json:"federationName"`
 	// zone name, like example.com.
 	ZoneName string `json:"zoneName"`
+	// suffix to use for publishing federated services.
+	FederationDnsSuffix string `json:"federationDnsSuffix"`
```

Comment:
nit: It's a suffix to the service name, but a prefix to the DNS zone.  Somewhat confusing, perhaps.  I'd suggest either adding an example to clarify, or renaming it something like DNSSubZone?